pkg/listener/metrics/cluster_policy_report.go

```diff
@@ -26,23 +26,23 @@ func CreateClusterPolicyReportMetricsListener(filter *report.ReportFilter) repor
 
 		switch event.Type {
 		case report.Added:
-			clusterPolicyGauge.WithLabelValues(newReport.Name, strings.Title(report.Skip)).Set(float64(newReport.Summary.Skip))
-			clusterPolicyGauge.WithLabelValues(newReport.Name, strings.Title(report.Pass)).Set(float64(newReport.Summary.Pass))
-			clusterPolicyGauge.WithLabelValues(newReport.Name, strings.Title(report.Warn)).Set(float64(newReport.Summary.Warn))
-			clusterPolicyGauge.WithLabelValues(newReport.Name, strings.Title(report.Fail)).Set(float64(newReport.Summary.Fail))
-			clusterPolicyGauge.WithLabelValues(newReport.Name, strings.Title(report.Error)).Set(float64(newReport.Summary.Error))
+			clusterPolicyGauge.WithLabelValues(newReport.Name, cases.Title(language.English, cases.NoLower).String(report.Skip)).Set(float64(newReport.Summary.Skip))
```

Comment:
You defines this multiple times instead of configure it at the top and reuse it

Reply:
Shared ones should be written in that method